Sources/Puree/Output/BufferedOutput.swift

```diff
@@ -1,10 +1,10 @@
 import Foundation
 
-open class BufferedOutput: Output {
+open class BufferedOutput: InstantiatableOutput {
```

Comment:
I think BufferedOutput must be InstantiatableOutput for the backward compatibility.